src/main/yamcs/etc/yamcs.oresat0.yaml

```diff
@@ -24,43 +24,61 @@ services:
 dataLinks:
   - name: beacon-tm-realtime
     class: org.yamcs.tctm.UdpTmDataLink
-    stream: tm_realtime
+    stream: beacon_tm_realtime
     port: 10015
     packetPreprocessorClassName: org.oresat.uniclogs.BeaconPacketPreprocessor
     packetPreprocessorArgs:
       timestampOffset: 26
 
   - name: beacon-tm-dump
     class: org.yamcs.tctm.UdpTmDataLink
-    stream: tm_dump
+    stream: beacon_tm_dump
     port: 10020
     packetPreprocessorClassName: org.oresat.uniclogs.BeaconPacketPreprocessor
     packetPreprocessorArgs:
       timestampOffset: 26
 
+
   - name: edl-tc-realtime
     class: org.yamcs.tctm.UdpTcDataLink
-    stream: tc_realtime
+    stream: edl_tc_realtime
     host: localhost
     port: 10025
     commandPostprocessorClassName: org.oresat.uniclogs.EDLCommandPostprocessor
 
+  - name: edl-tm-realtime
+    class: org.yamcs.tctm.UdpTmDataLink
+    stream: edl_tm_realtime
+    host: localhost
+    port: 10016
+    packetPreprocessorClassName: org.oresat.uniclogs.BeaconPacketPreprocessor
+    packetPreprocessorArgs:
+      timestampOffset: 26
+
 mdb:
   # Configuration of the active loaders
   # Valid loaders are: sheet, xtce or fully qualified name of the class
   - type: xtce
-    args:
-      file: mdb/oresat0-xtce.xml
+    spec: mdb/oresat0.xml
+    subLoaders:
+      - type: xtce
+        spec: mdb/oresat0/datatypes.xml
+      - type: xtce
+        spec: mdb/oresat0/parameters.xml
+      - type: xtce
+        spec: mdb/oresat0/commands.xml
+        
```

Comment:
As much as splitting these up into their own files would make XML more bearable, it's typically industry standard that there's one XML file per mission and that file represents all metadata for the Mission Database.

More discussion needed in review.